libs/design-system/src/lib/Components/SporksCard/index.tsx

```diff
@@ -49,24 +51,32 @@ const Spork = ({ heading, timestamp, sporkMetadata }) => {
     branchOrTag,
     dockerTag,
   } = sporkMetadata;
-  const [isExpanded, setIsExpanded] = useState(true);
+  const [isExpanded, setIsExpanded] = useState(isDefaultExpanded);
+  const cardStyles = clsx(
```

Comment:
Sometimes we use `clsx` sometimes we don't? What is the rule for deciding when to use and when not to use?